File: CS/DXGrid_AttributesBasedValidation/ViewModel.cs
using DevExpress.Mvvm;
using System.Collections.ObjectModel;

namespace DXGrid_AttributesBasedValidation {
    public class ViewModel : ViewModelBase {
        public ViewModel() {
            Source = Staff.GetStaff();
        }
        public ObservableCollection<Employee> Source { get; set; }
    }
}

File: VB/DXGrid_AttributesBasedValidation/ViewModel.vb
Imports DevExpress.Mvvm
Imports System.Collections.ObjectModel

Namespace DXGrid_AttributesBasedValidation

    Public Class ViewModel
        Inherits ViewModelBase

        Public Sub New()
            Source = GetStaff()
        End Sub

        Public Property Source As ObservableCollection(Of Employee)
    End Class
End Namespace
